erizo/src/erizo/DtlsTransport.cpp

```diff
@@ -242,24 +244,28 @@ void DtlsTransport::write(char* data, int len) {
 }
 
 void DtlsTransport::onDtlsPacket(DtlsSocketContext *ctx, const unsigned char* data, unsigned int len) {
-  int comp = 1;
+  packetPtr packet = std::make_shared<dataPacket>(1, data, len);
   if (ctx == dtlsRtcp.get()) {
```

Comment:
consider `packet->comp = ctx==dtlsRtcp.get() ? 2 : 1; rtcp_resender_>ScheduleResend(packet)`
or remove the `else` block and take `ScheduleResend` out